RecoMuon/L2MuonSeedGenerator/src/L2MuonSeedGeneratorFromL1T.cc

```diff
@@ -435,3 +977,82 @@ const TrajectorySeed* L2MuonSeedGeneratorFromL1T::associateOfflineSeedToL1( edm:
 
   return selOffseed;
 }
+
+
+bool L2MuonSeedGeneratorFromL1T::isAssociateOfflineSeedToL1( edm::Handle<edm::View<TrajectorySeed> > & offseeds,
+                                     std::vector< std::vector<double> > & dRmtx,
+                                     TrajectoryStateOnSurface & newTsos,
+                                     unsigned int imu,
+                                     std::vector< std::vector<const TrajectorySeed *> > & selOffseeds,
+                                     double dRcone ) {
+
+  const std::string metlabel = "Muon|RecoMuon|L2MuonSeedGeneratorFromL1T";
+  bool isAssociated = false;
+  MuonPatternRecoDumper debugtmp;
+
+  edm::View<TrajectorySeed>::const_iterator offseed, endOffseed = offseeds->end();
+  unsigned int nOffseed(0);
+
+  LogDebug(metlabel) << endl;
+  LogDebug(metlabel) << "[~ loop on L2 off seeds ~]" << endl;
+  LogDebug(metlabel) << endl;
+
+  for(offseed=offseeds->begin(); offseed!=endOffseed; ++offseed, ++nOffseed) {
+    LogDebug(metlabel) << "||||||||||||||||||||||||" << endl;
```

Comment:
There are tons of debug lines which make reading and understanding this mehod rather painfull.
Are really all them needed  (or where they just used by you while writing it from scratch)? Could they be better organized and compactified, perhaps only keeping the ones really useful for the code maintenance, and removing the ones only meant to your own initial dedug?